docs/_static/custom.css

```diff
@@ -152,3 +152,7 @@ img {
 .highlight .nn, .highlight .nc {
     text-decoration: none;
 }
+
+.shorturl {
+    overflow-x: auto;
+}
```

Comment:
Can we make it slightly smaller than the other text? Shouldn't be as big as normal text ;)

Reply:
Also what about centering the links?

Reply:
![sm-txt-cen](https://user-images.githubusercontent.com/29725587/46561732-928d5700-c916-11e8-914c-20ac052f321e.png)
Centering the links in small screen does not look good.

Reply:
I think it is better to have only one canonical shortlink, which makes the rendering easier.

The other ones will still work, but wouldnt be encouraged.

For this page, the best of the redirects is 'newcomer' , not the plural 'newcomers'

Also, we only need 'newcomer' to be rendered, as all shortlinks will be on domain `https://coala.io/` , so it is redundant to show the domain